pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -71,9 +71,14 @@ func (srv PrometheusSrv) RouteGetRuleStatuses(c *models.ReqContext) response.Res
 		namespaceUIDs = append(namespaceUIDs, k)
 	}
 
+	dashboardUID := c.Query("dashboard_uid")
```

Comment:
I think we want to do something similar also for [GET /api/ruler/grafana/api/v1/rules](https://github.com/grafana/grafana/blob/063160aae2949dc1c13d043099198e7ca31b90a2/pkg/services/ngalert/api/api_ruler.go#L150)